main.go

```diff
@@ -161,8 +159,16 @@ func main() {
 
 	initHttpProf(logger)
 
-	if emulatorIndexPath != "" {
-		fakeServer, err := initFakeGCSServer(logger, emulatorIndexPath)
+	if indexPath := os.Getenv("EPR_EMULATOR_INDEX_PATH"); indexPath != "" {
+		if !featureStorageIndexer {
+			logger.Fatal("EPR_EMULATOR_INDEX_PATH environment variable is set, but feature-storage-indexer is not enabled. Please enable it to use the fake GCS server.")
+		}
+		if storageIndexerBucketInternal != "" && storageIndexerBucketInternal != storage.FakeIndexerOptions.PackageStorageBucketInternal {
+			logger.Fatal("EPR_EMULATOR_INDEX_PATH environment variable is set, but storage-indexer-bucket-internal is already set to a different value. Please remove the flag or set it to the fake GCS server bucket " + storage.FakeIndexerOptions.PackageStorageBucketInternal)
+		}
+		// In this mode, the internal bucket is set to the fake GCS server bucket.
+		storageIndexerBucketInternal = storage.FakeIndexerOptions.PackageStorageBucketInternal
```

Comment:
Should we just ignore `-storage-indexer-bucket-internal` in this case? There is no use of this flag on this mode.
```suggestion
		// In this mode, the internal bucket is set to the fake GCS server bucket.
		storageIndexerBucketInternal = storage.FakeIndexerOptions.PackageStorageBucketInternal
```

Reply:
Not sure if that could be misleading for the users.

If they set the flag `-storage-indexer-bucket-internal` or the corresponding environment variable to another bucket, then they will not be able to access to the fake GCS server using the bucket they set in the flag/env var (e.g. query bucket objects or update the contents of the bucket to add a new index).

Maybe an option could be change the `logger.Fatal` by a `logger.Info` and rephrase the message. But even with that, the users would need to check the EPR logs to know why they cannot access to the bucket in the fake GCS server.